doc/sphinx/conf.py.in

```diff
@@ -6,46 +6,36 @@
 import shutil
 
 # The suffix of source filenames.
-from recommonmark.parser import CommonMarkParser
-from recommonmark.transform import AutoStructify
-
-# Assure symbolic link is removed
-if os.path.lexists('docs') and os.path.islink('docs'):
-    os.remove('docs')
-else:
-    shutil.rmtree('docs', ignore_errors=True)
-
-# Create symbolic link
-if os.path.exists('../../docs'):
-    os.symlink('../../docs', './docs')
-
-# Default doxygen location
-doxyDir = os.path.join(os.path.expanduser('~'),
-                           'cmake_builds',
-                           'cpt_sys_sw-fpga-sw',
-                           'doc',
-                           'xml')
-
-# Assure symbolic link is removed?
-if os.path.exists(doxyDir) and not os.path.exists('doxygen_xml'):
-    os.symlink(doxyDir, './doxygen_xml')
+# from recommonmark.parser import CommonMarkParser
+# from recommonmark.transform import AutoStructify
+
+# Another markdown parser for sphinx
+from sphinxcontrib.pandoc_markdown import MarkdownParser
 
 # Enabled extensions
 extensions = [
     'sphinx.ext.autodoc',
-    'sphinx.ext.intersphinx',
+    # 'sphinx.ext.intersphinx',
```

Comment:
Temporarily disabled? Or can be removed?

Reply:
can be removed will be removed